playbooks/roles/l2tp-ipsec/templates/instructions.md.j2

```diff
@@ -16,6 +16,46 @@ If the connection appears to time out, first check to make sure the passwords we
 
 <a name="windows"></a>
 ### Windows ###
+
+In order to follow the quick directions, you'll need to use Windows PowerShell ISE. To start it:
+
+1. Look in the *Start* menu, under *Windows PowerShell*.
+1. Right-click on *PowerShell ISE*, and choose "Run as administrator"
+
+#### Windows and NAT ####
+
+If you are connecting from behind a NAT (such as a home router), you need to add a setting to tell Windows to expect the NAT.
+
+Start PowerShell ISE. In the ISE command window, paste the following code, and press *Enter*.
+
+```
+Set-ItemProperty -Path HKLM:\SYSTEM\CurrentControlSet\Services\PolicyAgent -Name AssumeUDPEncapsulationContextOnSendRule -Value 2 -Type DWord
+```
+
+If you aren't using PowerShell, you could use `regedit` to add and set the value of `AssumeUDPEncapsulationContextOnSendRule`.
+
```

Comment:
I think the regedit instructions need to be more fleshed out (how do I navigate to the correct entry in regedit?). Alternatively, better to omit it entirely and could be made more thorough in an entry for the documentations folder, with pretty screenshots. 

Anyone else have thoughts on this?

Reply:
Isn't there a way to load a .reg fragment of some sort into regedit so you don't have to do GUI clicking stuff?

(I might be crazy, my Windows experience is suuuper dated/hazy)

Reply:
I'm punting on the `.reg` for now in the interest of getting this merged before further iteration. I totally need to go over the UI instructions on a couple Windows variants, and I don't have the necessary excitement to do that today. (Besides, the automatic setup method works better than the UI instructions. :-)